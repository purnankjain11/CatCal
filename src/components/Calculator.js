import React from 'react';
import {View} from 'react-native';
import DisplayBox from './DisplayBox';
import ButtonPanel from './ButtonPanel';

export default class Calculator extends React.Component {
  render() {
    return (<View>
      <DisplayBox />
      <ButtonPanel />
    </View>
  );
  }
}