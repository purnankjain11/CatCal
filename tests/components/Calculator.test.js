import React from 'react';
import Calculator from '../../src/components/Calculator';

import renderer from 'react-test-renderer';

describe('component snapshot test', () => {
  it('renders calculator without crashing', () => {
    const rendered = renderer.create(<Calculator />).toJSON();
    expect(rendered).toMatchSnapshot();
  });
});