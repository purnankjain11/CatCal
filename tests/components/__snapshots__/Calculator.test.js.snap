// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component snapshot test renders calculator without crashing 1`] = `
<View>
  <View>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
    />
  </View>
  <View />
</View>
`;
